ci/images/alpine/Dockerfile.php84

```diff
@@ -0,0 +1,71 @@
+# Base-image
+FROM php:8.4-alpine AS base
+
+ENV PATH=${PATH}:/usr/local/bin:/root/.local/bin
+
+RUN apk update && apk add --no-cache \
+    nodejs \
+    npm \
+    bash \
+    curl \
+    bzip2 \
+    git \
+    make \
+    python3 \
+    python3-dev \
+    py3-pip \
+    py3-virtualenv \
+    gcc \
+    g++ \
+    musl-dev \
+    openjdk21 \
+    icu-dev oniguruma-dev libxml2-dev libzip-dev curl-dev \
+    autoconf \
+    pkgconf \
+    re2c \
+    && npm install -g corepack \
+    && /usr/bin/python --version \
+    && /usr/bin/python -m pip install --break-system-packages --no-cache-dir --upgrade pipenv atom-tools --target /opt/pypi \
+    && docker-php-ext-install zip intl mbstring xml curl pdo pdo_mysql mysqli \
+    && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer \
+    && node -v \
+    && npm -v \
+    && php -v \
+    && composer --version \
+    && rm -rf /var/cache/apk/*
+
+
+# cdxgen-image
+FROM base AS cdxgen
+LABEL maintainer="CycloneDX" \
+    org.opencontainers.image.authors="Team AppThreat <cloud@appthreat.com>" \
+    org.opencontainers.image.source="https://github.com/CycloneDX/cdxgen" \
+    org.opencontainers.image.url="https://github.com/CycloneDX/cdxgen" \
+    org.opencontainers.image.version="rolling" \
+    org.opencontainers.image.vendor="CycloneDX" \
+    org.opencontainers.image.licenses="Apache-2.0" \
+    org.opencontainers.image.title="cdxgen" \
+    org.opencontainers.image.description="Rolling image with cdxgen SBOM generator for php apps" \
+    org.opencontainers.docker.cmd="docker run --rm -v /tmp:/tmp -p 9090:9090 -v $(pwd):/app:rw -t ghcr.io/cyclonedx/cdxgen-alpine-php84:v11 -r /app --server"
+
+ENV CDXGEN_IN_CONTAINER=true \
+    NODE_COMPILE_CACHE="/opt/cdxgen-node-cache" \
+    PYTHONPATH=/opt/pypi
+
+ENV PATH=${PATH}:/usr/local/bin:${PYTHONPATH}/bin:/opt/cdxgen/node_modules/.bin
+
+COPY . /opt/cdxgen
+
+RUN cd /opt/cdxgen && corepack enable && corepack pnpm install --config.strict-dep-builds=true --prod --package-import-method copy --frozen-lockfile && corepack pnpm cache delete \
+    && npm install -g --production @appthreat/atom-parsetools@1.0.4 \
+    && npm uninstall -g corepack \
+    && apk del npm py3-pip py3-virtualenv gcc musl-dev make curl tar bash g++ icu-dev oniguruma-dev libxml2-dev libzip-dev curl-dev \
+    autoconf pkgconf re2c \
+    && mkdir -p ${NODE_COMPILE_CACHE} \
+    && node /opt/cdxgen/bin/cdxgen.js --help \
+    && phpastgen --help \
```

Comment:
Add `atom-tools --help` in both alpine and debian, so that we know this command works.